[2024-12-25T21:20:15.869+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-25T21:20:15.883+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: instacart_etl.etl_fact_instacart manual__2024-12-25T14:50:19.065103+00:00 [queued]>
[2024-12-25T21:20:15.891+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: instacart_etl.etl_fact_instacart manual__2024-12-25T14:50:19.065103+00:00 [queued]>
[2024-12-25T21:20:15.892+0000] {taskinstance.py:2866} INFO - Starting attempt 10 of 10
[2024-12-25T21:20:15.906+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): etl_fact_instacart> on 2024-12-25 14:50:19.065103+00:00
[2024-12-25T21:20:15.915+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1979) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-25T21:20:15.916+0000] {standard_task_runner.py:72} INFO - Started process 1981 to run task
[2024-12-25T21:20:15.915+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'instacart_etl', 'etl_fact_instacart', 'manual__2024-12-25T14:50:19.065103+00:00', '--job-id', '235', '--raw', '--subdir', 'DAGS_FOLDER/instacart_etl_dag.py', '--cfg-path', '/tmp/tmpd9ydb__a']
[2024-12-25T21:20:15.917+0000] {standard_task_runner.py:105} INFO - Job 235: Subtask etl_fact_instacart
[2024-12-25T21:20:15.962+0000] {task_command.py:467} INFO - Running <TaskInstance: instacart_etl.etl_fact_instacart manual__2024-12-25T14:50:19.065103+00:00 [running]> on host 53a83a5c1c65
[2024-12-25T21:20:16.040+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='instacart_etl' AIRFLOW_CTX_TASK_ID='etl_fact_instacart' AIRFLOW_CTX_EXECUTION_DATE='2024-12-25T14:50:19.065103+00:00' AIRFLOW_CTX_TRY_NUMBER='10' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-25T14:50:19.065103+00:00'
[2024-12-25T21:20:16.041+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-25T21:20:16.498+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-25T21:20:16.506+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-25T21:20:16.507+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=instacart_etl, task_id=etl_fact_instacart, run_id=manual__2024-12-25T14:50:19.065103+00:00, execution_date=20241225T145019, start_date=20241225T212015, end_date=20241225T212016
[2024-12-25T21:20:16.532+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-25T21:20:16.545+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-25T21:20:16.551+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
